float64[6] l_arm_p
float64[6] r_arm_p